frontend/src/components/AppointmentCard.tsx

```diff
@@ -0,0 +1,38 @@
+import './AppointmentCard.scss';
+import {useContext, useEffect, useState} from "react";
+import axios from "axios";
+import {AuthContext} from "../context/AuthProvider";
+
+
+export default function AppointmentCard() {
+
+    const [appointments, setAppointments] = useState([])
+
+    const {token} = useContext(AuthContext)
+
+    useEffect(() => {
+        axios.get("/api/appointments", token? {
+            headers: {
+                "Authorization": token
+            }
+        } : {})
+            .then(response => {
+                console.log(response)
+                setAppointments(response.data)
+            })
+            .catch(console.error)
+    }, [])
+
+    return (
+        <div className="card">
+            {appointments?.map((appointment: any) => (
+                <div key={appointment.username}>
```

Comment:
Can a user only have one appointment? Maybe we should use an id here?

Reply:
Ups, you're right. specified any as type so that IntelliJ is quiet for the moment